src/resources/LimitedRestorationExtension/ULC.ts

```diff
@@ -8,19 +8,11 @@ export const UlcRestorationResource: ResourceIF = {
   displayName: GetCorpFullDescription(CorpTypeCd.BC_ULC_COMPANY),
   addressLabel: 'Registered Office',
   showEmailUnderName: false,
-  filingData: [
-    // order matters - see restorationResource()
-    {
-      entityType: CorpTypeCd.BC_ULC_COMPANY,
-      filingTypeCode: FilingCodes.RESTORATION_LTD_EXTEND,
-      priority: false
-    },
-    {
-      entityType: CorpTypeCd.BC_ULC_COMPANY,
-      filingTypeCode: FilingCodes.RESTORATION_LTD_TO_FULL,
-      priority: false
-    }
-  ],
+  filingData: {
+    entityType: CorpTypeCd.BC_COMPANY,
+    filingTypeCode: FilingCodes.RESTORATION_LTD_EXTEND,
+    priority: false
+  },
   changeData: {
     nameChangeOptions: [
```

Comment:
I believe Limited Restoration Extension doesn't allow changing the business name, so you can leave this object out (also in files above).

Reply:
Apologies, it's late in the day.  I'm not sure I understand what you mean.  Should I remove the following attribute from all the `src/resources/LimitedRestorationExtension/*` resource files?

```javascript
  filingData: {
    entityType: null,
    filingTypeCode: FilingCodes.RESTORATION_LTD_EXTEND,
    priority: false
  },
```

And, the following line from the `LimitedRestorationExtension.vue` view?

```javascript
this.setFilingData([this.restorationResource.filingData])
```

Reply:
I believe you can remove the `nameChangeOptions` object from the resource files for Limited Restoration Extensions.